magstock/templates/regextra.html

```diff
@@ -41,8 +43,8 @@
         if ($.field('badge_printed_name')) {
             $.field('badge_printed_name').parents('.form-group').remove();
         }
-        {% if c.PAGE_PATH != '/registration/form' %}
-            $.field('amount_extra').parents('.form-group').hide();
+        {% if c.PAGE_PATH == '/registration/form' %}
+            $('#vehicle-warning').hide();
```

Comment:
This was an unrelated, minor change that hides the new vehicle warning text while you're on the admin page, because it just takes up space.